EdgeOfEmpireBot/EdgeOfEmpireBot/MessageHandlers/SteamHandler/SteamMessageHandler.cs

```diff
@@ -0,0 +1,39 @@
+using HK47.MessageHandlers.Interfaces;
+using HK47.Services;
+
+namespace HK47.MessageHandlers;
+
+public class SteamMessageHandler(ISteamService steamService, IDataService dataService, IMessageService messageService) : ISteamMessageHandler
+{
+    /// </inheritdoc>
+    public async Task ProcessCommand(string userInput)
+    {
+        var commandParams = userInput.Split(' ');
+        var command = commandParams!.First().ToLower();
+        switch (command)
+        {
+            case "requestgame":
+            case "gr":
+                try
```

Comment:
We should probably at some point clean this up to callign methods for readablty and cognitive complexity

Reply:
yeah make it an issue and we will plan for future release